playbooks/provisioning/openstack/provision-openstack.yml

```diff
@@ -23,6 +23,7 @@
     num_nodes: "{{ openstack_num_nodes }}"
     num_infra: "{{ openstack_num_infra }}"
     num_dns: "{{ openstack_num_dns | default(1) }}"
+    node_removal_policies: "{{ openstack_node_removal_policies | to_yaml }}"
```

Comment:
can you put `"{{ openstack_node_removal_policies | default([]) | to_yaml }}"` here?

Without the `default` filter, everyone has to put this in their inventory even if they don't use it.